test/bad_dtls_test.c

```diff
@@ -548,8 +548,9 @@ static int test_bad_dtls(void)
 
         if (!TEST_true(send_record(rbio, SSL3_RT_APPLICATION_DATA, tests[i].seq,
                                    &tests[i].seq, sizeof(uint64_t)))) {
-            TEST_error("Failed to send data seq #0x%lx (%d)\n",
-                       tests[i].seq, i);
+            TEST_error("Failed to send data seq #0x%08x%08x (%d)\n",
+                       (uint32_t)(tests[i].seq >> 32),
+                       (uint32_t)(tests[i].seq & 0xFFFFFFFF), i);
```

Comment:
Also, PRINT_FORMAT could have a bit more guarding.  Have a look at the guards around the definition of `__bio_h__attr__` in `include/openssl/bio.h`